cmdapp/utils/extension.py

```diff
@@ -27,31 +27,31 @@ def merge(d1: dict, d2: dict):
                 result[k] = d2[k]
         return result | {k: v for k, v in d2.items() if k not in result}
 
-    def filter(d: dict, *args, **kwargs) -> dict:
-        """Filter items with keys in `args` and rename items with keys in `kwargs`
+    def filter(d: dict, *args, rename: dict[str, str] = {}) -> dict:
```

Comment:
Move from using `kwargs` to `dict` argument support a non-identifier rename key, like a dict `{"special key with space": "value"}` can be renamed